src/Symfony/Bundle/WebProfilerBundle/Twig/WebProfilerExtension.php

```diff
@@ -116,4 +116,49 @@ public function getName()
     {
         return 'profiler';
     }
+
+    /**
+     * @see \Twig\Extension\EscaperExtension::escape()
+     */
+    private static function escape(string $s, string $charset): string
```

Comment:
I would also use the Twig internal method instead.

Reply:
done